src/components/PopupConfirm.js

```diff
@@ -0,0 +1,23 @@
+import Popup from "./Popup.js";
+import { selectors } from "../utils/constants.js";
+
+export default class PopupConfirm extends Popup {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.